_includes/manuals/1.11/2.1_quickstart_installation.md

```diff
@@ -147,7 +147,8 @@ yum -y install http://yum.theforeman.org/releases/{{page.version}}/f21/x86_64/fo
 {% highlight bash %}
 echo "deb http://deb.theforeman.org/ jessie {{page.version}}" > /etc/apt/sources.list.d/foreman.list
 echo "deb http://deb.theforeman.org/ plugins {{page.version}}" >> /etc/apt/sources.list.d/foreman.list
-wget -q http://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
+apt-get -y install ca-certificates
+wget -q https://deb.theforeman.org/pubkey.gpg -O- | apt-key add -
```

Comment:
This won't work out of the box for Debian - https support isn;t enabled by default in apt (sigh, i know):

```
greg@topaz:~$ cat /etc/apt/sources.list.d/foreman.list 
deb https://deb.theforeman.org/ jessie 1.11
deb https://deb.theforeman.org/ plugins 1.11
greg@topaz:~$ sudo apt-get update
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
```
It's probably enough to add apt-transport-https to the preceeding line (and ditto all the ones below)